torch/nn/functional.py

```diff
@@ -1228,6 +1228,10 @@ def adaptive_avg_pool2d(input: Tensor, output_size: BroadcastingList2[int]) -> T
     """
     if has_torch_function_unary(input):
         return handle_torch_function(adaptive_avg_pool2d, (input,), input, output_size)
+    if not isinstance(output_size, int):
+        if len(output_size)!=2:
+            raise ValueError(f"output_size must be type int (for square output)" 
```

Comment:
I think you don't need the f here
```suggestion
            raise ValueError("output_size must be of type int (for square output)" 
```